### PR TITLE
fix(select-dropdown): remove min-width at transparent style type

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [x] Apps
  - [x] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
- Console
  image

- Mirinae
  image

### CODE REVIEW

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 1]
```diff
@@ -339,6 +339,7 @@ export default defineComponent<SelectDropdownProps>({
         }
     }
     &.transparent {
+        min-width: unset;
```
Comment (WANZARGEN) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:342]: question: Is there any side effects? 🫣

**Review by WANZARGEN — APPROVED** [review]
Very simple :) LGTM

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue (hunk 1 above)
Comment (skdud4659) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:342]: I checked it at console!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Pull request merged**